README.md

```diff
@@ -31,31 +31,27 @@ After adding the snapshots repository, you can add the `google-cloud-graalvm-sup
 
 This dependency contains the GraalVM configurations (reflection config, native call config) to provide out-of-the-box support for native-image compilation of applications depending on the Google Java Client Libraries.
 
-### Note on grpc-netty-shaded version
+### Client Library Versions
 
-As an additional step, you will also need to override the version of `grpc-netty-shaded` to version `1.32.1` (or later):
+To get your application working with GraalVM compilation, you will also have to ensure that the client library versions in your project are supported by your chosen version of `google-cloud-graalvm-support`.
 
-```
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>io.grpc</groupId>
-        <artifactId>grpc-netty-shaded</artifactId>
-        <version>1.32.1</version>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-```
+| GraalVM Support version | *`libraries-bom` version | `grpc-netty-shaded` version |
+|-------------------------|:-------------------------|-----------------------------|
+| `1.0.0-SNAPSHOT`        | `>= 11.0.0`              | `>= 1.32.1`                 |
 
-The Java client libraries import this dependency to your project automatically as a transitive dependency.
-The latest version of `grpc-netty-shaded` contains a patch to enable native image compilation, so this version must be (temporarily) manually upgraded for native image compilation to work.
+**NOTE:** Most users typically manage their client library versions using the [Cloud Libraries Bill of Materials](https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM) (`libraries-bom`).
+The `libraries-bom` manages the version of `grpc-netty-shaded` for you as well so you don't have to manage it yourself.
 
-## Samples
+## Supported Libraries
 
-Sample projects can be found in the [google-cloud-graalvm-samples](https://github.com/GoogleCloudPlatform/google-cloud-graalvm-support/tree/master/google-cloud-graalvm-samples) directory.
+This project should support many Java Google Client Libraries out-of-the-box.
 
-## Supported Libraries
+Our project has [code samples](https://github.com/GoogleCloudPlatform/google-cloud-graalvm-support/tree/master/google-cloud-graalvm-samples) demonstrating compatibility with the following APIs:
```

Comment:
I think you could just use a relative link, no? `./google-cloud-graalvm-samples`?

Reply:
Done. Cool didn't know about this feature!